lib/drivers/mock/mock.go

```diff
@@ -6,21 +6,32 @@ import (
 
 type MockDriver struct {
 	Revisions []core.Revision
-	State     core.Revision
 }
 
 func (this *MockDriver) Init(config map[string]string) error {
 	return nil
 }
 
-func Migrate(r core.Revision, ec chan error) {
+func (this *MockDriver) Migrate(r core.Revision, ec chan error) {
```

Comment:
Same for migrate, you don't need to apply the migrations but simply append the Revision to the end of the array and push nil to the ec